posts/inside-rust/2020-02-27-pietro-joins-core-team.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "Pietro Albini has joined the core team"
+author: Nick Cameron
+team: Core Team
+---
+
+We are very happy to announce that Pietro Albini @pietroalbini has joined the core team. Pietro joined us back on December 24th 2019 (a Christmas present for the core team!), but we have been a bit late in announcing it (sorry Pietro!).
+
+Pietro is the co-leader of the infrastructure team, and a member of the crates.io, docs.rs, and release teams, and the security response and Rustfix working groups. He has been instrumental in setting up and maintaining the various services and technology which keeps the Rust project moving quickly and smoothly.
```

Comment:
I'd remove the mention of rustfix, I didn't really do much there. Also I guess I'm also co-leading docs.rs now.